### PR TITLE
Create dotnet-desktop.yml

### PR DESCRIPTION
**Pull request by kozaka-tv**
This is an action, what builds here in github the develop branch.

### CODE REVIEW

**Review by Th0rstenf — APPROVED** [review]

File: .github/workflows/dotnet-desktop.yml [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+      run: dotnet test
+
+    # Restore the application to populate the obj folder with RuntimeIdentifiers
+    - name: Restore the application
+      run: msbuild $env:Solution_Name /t:Restore /p:Configuration=$env:Configuration
+      env:
+        Configuration: ${{ matrix.configuration }}
+
+    # Decode the base 64 encoded pfx and save the Signing_Certificate
+    - name: Decode the pfx
```
Comment (Th0rstenf) [inline .github/workflows/dotnet-desktop.yml:91]: This part is currently failing. and i'm not sure If i'll find the time to have a look at this signing process anytime soon

Comment (kozaka-tv) [reply .github/workflows/dotnet-desktop.yml:91]: I just created from the Github template. Could not test, so maybe it is enough to comment out not working parts with TODO just to have something.

Comment (kozaka-tv) [reply .github/workflows/dotnet-desktop.yml:91]: https://github.com/marketplace/actions/code-sign-a-file-with-pfx-certificate

Comment (kozaka-tv) [reply .github/workflows/dotnet-desktop.yml:91]: 2. Signing
# Generate a signing certificate in the Windows Application
# Packaging Project or add an existing signing certificate to the project.
# Next, use PowerShell to encode the .pfx file using Base64 encoding
# by running the following Powershell script to generate the output string:
#
# $pfx_cert = Get-Content '.\SigningCertificate.pfx' -Encoding Byte
# [System.Convert]::ToBase64String($pfx_cert) | Out-File 'SigningCertificate_Encoded.txt'
#
# Open the output file, SigningCertificate_Encoded.txt, and copy the
# string inside. Then, add the string to the repo as a GitHub secret
# and name it "Base64_Encoded_Pfx."
# For more information on how to configure your signing certificate for
# this workflow, refer to https://github.com/microsoft/github-actions-for-desktop-apps#signing
#
# Finally, add the signing certificate password to the repo as a secret and name it "Pfx_Key".
# See "Build the Windows Application Packaging project" below to see how the secret is used.

**Review by Th0rstenf — APPROVED** [review]

File: .github/workflows/dotnet-desktop.yml (hunk 1 above)
Comment (kozaka-tv) [reply .github/workflows/dotnet-desktop.yml]: I commented out this signing stuff. It is good now till run tests. Idk...hopefully this is a good base to CI/CD

**Comment by kozaka-tv** [comment]
Working :)

image

**Pull request merged**